docs/docs/whether.md

```diff
@@ -0,0 +1,104 @@
+your code, JSpecify annotations will improve the null-safety of Kotlin code
+using your Java library. New versions of the Kotlin compiler
+[recognize and correctly interpret](#kotlin) the annotations, so JSpecify
+annotated Java code will have correct nullness typing in Kotlin.
+
+## Nullness checker support for JSpecify
+
+Several major Java nullness analyzers already support JSpecify annotations to
+some degree. Consult the official documentation for your checker to see what
+they claim about their current and planned JSpecify support.
+
+*   The [EISOP Framework](https://eisop.github.io/) has good conformance except
+    for its interpretation of unspecified-nullness code (unannotated code
+    outside of `@NullMarked` scope).
+
+*   [NullAway](https://github.com/uber/NullAway) supports JSpecify annotations
+    but does not yet analyze generics.
+
+*   The [Checker Framework](https://checkerframework.org/) understands
+    `@Nullable` and `@NonNull`, but not `@NullMarked` or `@NullUnmarked`.
+
+*   JSpecify’s
+    [reference checker](https://github.com/jspecify/jspecify-reference-checker)
+    is mostly correct except for a few edge cases. While you can try it out, it
+    is not intended for production use and is not intended to be user-friendly
+    or performant.
+
+We are working on a
+[conformance test suite](https://github.com/jspecify/jspecify/tree/main/conformance-tests)
+that will allow any nullness analyzer to measure and publish how well they
+conform to the JSpecify specification. At this time, only JSpecify’s reference
+checker and the EISOP Framework have integrated into our prerelease test suite.
+
+## Kotlin
+
+Kotlin has a null-safe type system that’s similar to JSpecify’s model. But the
+Kotlin compiler interprets unannotated Java dependencies as having
+[platform types](https://kotlinlang.org/docs/java-interop.html#null-safety-and-platform-types),
+relaxing some of the nullness checks Kotlin otherwise performs. However, the
+compiler understands JSpecify annotations and is able to use them to make Kotlin
+code see null-safe types when calling into Java code. The Kotlin compiler
+correctly interprets `@Nullable` and `@NullMarked` starting at version 1.8.20,
+`@NonNull` starting at 2.0.0, and `@NullUnmarked` coming in 2.0.20.
+
+Note that the Kotlin compiler currently emits only warnings by default for
+problems found using JSpecify nullness. To change those to errors, pass the
+[`-Xjspecify-annotations=strict` and
+`-Xtype-enhancement-improvements-strict-mode`
+flags](https://kotlinlang.org/docs/whatsnew1520.html#support-for-jspecify-nullness-annotations).
+The Kotlin team plans to emit errors by default in Kotlin 2.1.0.
+
+## Annotation processors
+
+If your project relies on annotation processors, like [Dagger], that interpret
+nullness annotations on symbols in the classpath, then you may need to wait to
+adopt JSpecify annotations until you can build with JDK 22. There was a bug in
```

Comment:
I think we moved the "build with JDK 22" link (currently on line 96 below) up to here in the draft doc, so let's do it in the Markdown, too.